### PR TITLE
Add runtime dependency libiconv

### PR DESCRIPTION
**Pull request by dfornika**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by dfornika** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ requirements:
   run:
     - zlib
     - popt 1.16
+    - libiconv
```
Comment (bgruening) [inline recipe/meta.yaml]: @dfornika that is not the issue. The issue is that pinning to popt. popt depends on libiconv, but this old version of popt has not pinned libiconv ... so using a newer popt version would be the correct fix I think.

Comment (bgruening) [reply recipe/meta.yaml]: popt has, as well as zlib, run_exports defined. So in theory you can remove the pinning from popt host dependency and remove all run-dependencies.

Comment (dfornika) [reply recipe/meta.yaml]: Ok, thanks for that insight. I'll close this PR.

Comment (bgruening) [reply recipe/meta.yaml]: Oh no please reopen ... I will try to modify it ...

**Pull request closed without merging**